pkg/daemon/server/service/health_status.go

```diff
@@ -59,11 +59,11 @@ const (
 )
 
 // HealthThresholds are the thresholds used to compute the health of a vertex
-const (
+var (
 	// criticalBufferThreshold is the threshold above which the health of a vertex is critical
-	criticalBufferThreshold = 95
+	criticalBufferThreshold = v1alpha1.DefaultBufferUsageLimit * float64(90) // 95
```

Comment:
The threshold for different vertex should be different - based on the vertex `limits` configuration, is this right? @kohlisid

Reply:
Yes now that we have it relative, it can vary vertex to vertex. 
The code will need changes to compare each timeline using the limits for the corresponding vertex limits.